src/controllers/server.js

```diff
@@ -615,26 +615,24 @@ class Server extends EventEmitter {
             // and check the final path resoltuion for that directory. If there is an unexpected error or
             // we get to a point where the path is now shorter than the data path we will exit and return
             // the base data path.
-            _.some(pathParts, () => {
```

Comment:
`_.some` stops iteration once `true` is returned, `_.forEach` when `false` is returned. I see no advantage.

Reply:
https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/collection-method-value.md

Reply:
Ah, all right.